cam0:
  cam_overlaps: [1, 2, 3]
  camera_model: pinhole
  distortion_coeffs: [0.8993594340759864]
  distortion_model: fov
  intrinsics: [680.7351521518964, 681.3434442709168, 654.2813663973375, 518.9859071696467]
  resolution: [1280, 1024]
  rostopic: /cam/global/left/image_raw
cam1:
  T_cn_cnm1:
  - [0.9999742736479438, -0.0033013978764068405, 0.006368109164565337, -0.0999617280562253]
  - [0.003399202856852304, 0.999875490528827, -0.015409375706969089, -0.0006856790450085342]
  - [-0.006316443794425133, 0.0154306257748096, 0.9998609895010369, 0.0008400790120027251]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0, 2, 3]
  camera_model: pinhole
  distortion_coeffs: [0.898565297193209]
  distortion_model: fov
  intrinsics: [683.5030383207096, 684.465584180249, 630.4903687076826, 517.5419678549986]
  resolution: [1280, 1024]
  rostopic: /cam/global/right/image_raw
cam2:
  T_cn_cnm1:
  - [0.9999659143893593, 0.0018113700518159706, -0.0080553707543065, 0.09993875205841854]
  - [-0.001571677690341415, 0.9995587202160532, 0.029663018546415707, -0.057665566648060294]
  - [0.00810554678548186, -0.029649347017813115, 0.9995274965366021, 0.0016920507507299217]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0, 1, 3]
  camera_model: pinhole
  distortion_coeffs: [0.8983574862363564]
  distortion_model: fov
  intrinsics: [688.0754364916569, 689.0032036918133, 620.6408646353625, 518.3245046638706]
  resolution: [1280, 1024]
  rostopic: /cam/rolling/left/image_raw
cam3:
  T_cn_cnm1:
  - [0.9999790730920792, -0.0006410587482835392, -0.006437578860529536, -0.10132764770321892]
  - [0.00040277502422118844, 0.9993171157138827, -0.0369478012006752, -0.0008943141850341475]
  - [0.006456868450274579, 0.03694443510146016, 0.9992964612990722, -0.0014310576711502548]
  - [0.0, 0.0, 0.0, 1.0]
  cam_overlaps: [0, 1, 2]
  camera_model: pinhole
  distortion_coeffs: [0.8984921645657551]
  distortion_model: fov
  intrinsics: [681.7575965394548, 682.8614723939833, 624.6721481158661, 498.03246676949]
  resolution: [1280, 1024]
  rostopic: /cam/rolling/right/image_raw
